src/templates/testmoderibbon.html

```diff
@@ -0,0 +1,9 @@
+<div class="ribbon-wrapper">
```

Comment:
Could we prefix the class name with `bookingjs-`? Then it's the same as other top-level DOM nodes, and it helps with CSS specificity (remember that the widget lives directly on other people's sites)

In general I think we should strive for following BEM naming of CSS classes, or in bookingjs case a bit modified to fit the existing names, i.e.
```
.bookingjs-ribbon
  .bookingjs-ribbon-wrap
    .bookingjs-ribbon-wrap-inner
```

![screen shot 2017-09-26 at 14 25 09](https://user-images.githubusercontent.com/222419/30860232-c0e7c558-a2c6-11e7-944e-a49b1f732793.png)

Reply:
yes of course